### ISSUE TITLE
Font Boldness

### ISSUE THREAD

**Issue by georgitrifonov** [Eugeny/tabby#9338]
Version: 1.0.205
Platform: macOS arm64 23.2.0
Plugins: highlight
Frontend: xterm-webgl

**Description:**
The font rendering in Tabby Terminal appears to be bolder compared to iTerm2, leading to a noticeable difference in the visual appearance of text.
This inconsistency may affect the overall user experience, especially for those transitioning between terminals.

<img width="653" alt="image" src="https://github.com/Eugeny/tabby/assets/42679919/6cfe9354-2364-487b-8714-090791e4c708">

My tabby Appearance:
![image](https://github.com/Eugeny/tabby/assets/42679919/8b890aa5-28c1-4d46-964e-d85f99df06a4)

**Comment by Chuckinator2020** [comment]
Have you tried lowering the "Normal font weight"?

**Comment by georgitrifonov** [comment]
> Have you tried lowering the "Normal font weight"?

Yes, I did, but unfortunately without effect. The text weight doesn't seem to change.
